lib/resources/pip.rb

```diff
@@ -15,10 +15,16 @@ class PipPackage < Inspec.resource(1)
       describe pip('Jinja2') do
         it { should be_installed }
       end
+
+      describe pip('django', '/path/to/virtualenv/bin/pip') do
+        it { should be_installed }
+        its('version') { should eq('1.11.4')}
+      end
     "
 
-    def initialize(package_name)
+    def initialize(package_name, virtualenv_path=nil)
```

Comment:
[nit] I think `virtualenv_path` is a bit confusing here: From looking at this, I'd expect that I was to put `/path/to/virtualenv` in there, not `/path/to/virtualenv/bin/pip`. No idea what's the best approach, though.